mcf-browser/src/back-end/test/parse-tmcf.test.ts

```diff
@@ -43,32 +43,6 @@ test('testing fillPropertyValues', () => {
   expect(filledPropVals2).toBe(expected2);
 });
 
-test('testing fillTemplateFromRow', () => {
-  const parser = new ParseTmcf();
-  parser.csvIndex = 8;
-  const filledTemp =
-      parser.fillTemplateFromRow(TestStr.testTMCF2, TestStr.testCSV2[0]);
-  expect(filledTemp).toBe(TestStr.expectedFilledTemp0);
-
-  // testing multiple propValues that are comma separated
-  const template =
-      'Node: dcid:test1\npropLabel1: C:TestSet->Col1, C:TestSet->Col2';
-  const row = {
-    'Col1': 'dcid:propVal1',
-    'Col2': 'dcid:propVal2',
-  };
-  const expectedMCF =
-      'Node: dcid:test1\npropLabel1: dcid:propVal1, dcid:propVal2';
-  const filledTemp2 = parser.fillTemplateFromRow(template, row);
-  expect(filledTemp2).toBe(expectedMCF);
-});
-
-test('testing csvToMCF', () => {
-  const parser = new ParseTmcf();
-  const mcf = parser.csvToMcf(TestStr.testTMCF1, TestStr.testCSV1);
-  expect(mcf).toBe(TestStr.expectedMCF1);
-});
-
```

Comment:
For my own understanding: why are these tests being removed?

Reply:
These are test cases for functions that no longer exist in the rewritten parse-tmcf file. The functions were essentially duplicates of previous parsing logic and now that the whole process is rewritten to do everything in one pass, the tests are useless.